### DISCUSSION TITLE
Play商店里的Chash Meta For Android不更新了么?

### DISCUSSION THREAD

**Discussion by Rangtian** [MetaCubeX/ClashMetaForAndroid#356] [General]
还停留在去年11月的2.9.0版本

**Comment by fejelaqbhbv** [comment]
从来就没上架过play store
那是有人自行编译上传的

官方渠道只有 https://github.com/MetaCubeX/ClashMetaForAndroid/releases 和 https://f-droid.org/packages/com.github.metacubex.clash.meta/

**Comment by dreamoeu** [comment]
> 还停留在去年11月的2.9.0版本

自己去联系play store的发布者

**Comment by caliban511** [comment]
何止， F-Droid 貌似也没更新了，，，
